### DISCUSSION TITLE
Doubt regarding correspondence with original MAW code for Incremental Clustering

### DISCUSSION THREAD

**Discussion by Anurag19101996** [uw-ssec/MAWpy#39] [General]
Also in line 117, we are checking if the row entry’s lat and long are both -1, and then we add the row entry in line 127 as part of the stay cluster. Could you explain in which lines in the original code are the reference for these executions from line 117-127? This is less of a question but clarification, to make sure we understand the code clearly and cross check with the original code. 

@qzchen-uw @gracejia513 @anujsinha3 @carlosgjs

**Comment by anujsinha3** [comment]
The code at line [117](https://github.com/uw-ssec/MAWpy/blob/main/src/mawpy/IncrementalClustering.py#L322) relates to the following 'else' [line](https://github.com/uw-ssec/MAWpy/blob/main/src/mawpy/IncrementalClustering.py#L322) of old code.

The corresponding ['if'](https://github.com/uw-ssec/MAWpy/blob/main/src/mawpy/IncrementalClustering.py#L295) in the old code.
